sentry.go

```diff
@@ -79,14 +85,29 @@ func (so *SentryOutput) getClient(dsn string) (client *raven.Client, err error)
 
 func (so *SentryOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (err error) {
 	var (
-		e      error
-		pack   *pipeline.PipelinePack
-		client *raven.Client
+		e        error
+		pack     *pipeline.PipelinePack
+		client   *raven.Client
+		contents []byte
 	)
 
+	encoder := or.Encoder()
+	if encoder == nil {
+		payloadEncoder := new(plugins.PayloadEncoder)
+		payloadEncoder.Init(new(plugins.PayloadEncoderConfig))
+		encoder = payloadEncoder
+	}
```

Comment:
There's no need to maintain backwards compatibility, when encoders were added it was clearly announced as a breaking change, and a number of outputs were changed to require an encoder.